Framework/Kernel/src/TimeSeriesProperty.cpp

```diff
@@ -33,6 +33,20 @@ TimeSeriesProperty<TYPE>::TimeSeriesProperty(const std::string &name)
     : Property(name, typeid(std::vector<TimeValueUnit<TYPE>>)), m_values(),
       m_size(), m_propSortedFlag(), m_filterApplied() {}
 
+/**
+ * Constructor
+ * @param name :: The name to assign to the property
+ * @param times :: A vector of DateAndTime objects
+ * @param values :: A vector of TYPE
+ */
+template <typename TYPE>
+TimeSeriesProperty<TYPE>::TimeSeriesProperty(
```

Comment:
This is a good addition to the code. It cleans up case of default construction plus set values. Could we add a unit test for this?

Reply:
Will do